src/proof/nova.rs

```diff
@@ -1472,72 +1472,70 @@ mod tests {
 
     #[test]
     #[ignore]
-    fn outer_prove_evaluate_zero_arg_lambda1() {
+    fn outer_prove_evaluate_zero_arg_lambda() {
         let s = &mut Store::<Fr>::default();
         let expected = s.num(123);
         let terminal = s.get_cont_terminal();
         nova_test_aux(
             s,
-            "((lambda () 123))",
+            "((lambda(x) x) 123)",
             Some(expected),
             None,
             Some(terminal),
             None,
-            3,
+            4,
         );
     }
 
     #[test]
     #[ignore]
     fn outer_prove_evaluate_zero_arg_lambda2() {
         let s = &mut Store::<Fr>::default();
-        let expected = s.num(10);
+        let expected = s.num(123);
         let terminal = s.get_cont_terminal();
         nova_test_aux(
             s,
-            "(let ((x 9) (f (lambda () (+ x 1)))) (f))",
+            "((lambda (y) ((lambda (x) y) 321)) 123)",
```

Comment:
This doesn't seem to be an equivalent test. I don't think we should be removing test cases. If you want to add test cases for consistency with other tests, that's fine — but please let's not remove. It would not be good if we decreased coverage and introduced regressions when refactoring/optimizing/extending the circuit.